src/pcp2opentelemetry/pcp2opentelemetry.py

```diff
@@ -0,0 +1,675 @@
+#!/usr/bin/env pmpython
+#
+# Copyright (C) 2025 Lauren Chilton <lchilton@redhat.com>
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License as published by the
+# Free Software Foundation; either version 2 of the License, or (at your
+# option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+# or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+# for more details.
+
+
+""" PCP to OPENTELEMETRY Bridge """
+
+# Common imports
+from collections import OrderedDict
+import errno
+import time
+import sys
+import platform
+
+# Our imports
+import requests
+import os
+import cpmapi
+import json
+
+# PCP Python PMAPI
+from pcp import pmapi, pmconfig
+from cpmapi import PM_CONTEXT_ARCHIVE, PM_INDOM_NULL, PM_DEBUG_APPL1, PM_TIME_SEC
```

Comment:
PM_DEBUG_APPL1 not available on some machines.  Looking at the pcp2openmetrics it eliminates this and just uses "appl1" in place of PM_DEBUG_APPL1 in the execute method.